### PR TITLE
Show that ServicePulse is running with a trial license

### PR DESCRIPTION
**Pull request by ramonsmits**
With trial license:

With trial license

Without:

Without trial license

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@ramonsmits does the screenshot attached still represent how it works?

**Review by SzymonPobiega — APPROVED** [review]

**Comment by ramonsmits** [comment]
> @ramonsmits does the screenshot attached still represent how it works?

@SzymonPobiega yes, that was updated to reflect current state.

**Review by tmasternak — APPROVED** [review]

**Comment by tmasternak** [comment]
@ramonsmits should we try writing tests for this?

File: src/Frontend/test/preconditions/hasLicense.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ const createLicenseMockedResponse =
         edition: "Enterprise",
         expiration_date: "",
         upgrade_protection_expiration: "2050-01-01T00:00:00.0000000Z",
-        license_type: "Commercial",
+        license_type: status == LicenseStatus.ValidWithExpiringTrial || status == LicenseStatus.InvalidDueToExpiredTrial ?  "Trial": "Subscription",
```
Comment (ramonsmits) [inline src/Frontend/test/preconditions/hasLicense.ts:48]:
```suggestion
        license_type: status == LicenseStatus.ValidWithExpiringTrial || status == LicenseStatus.InvalidDueToExpiredTrial ?  "Trial": "Commercial",
```

File: src/Frontend/test/specs/licensing/trial-license-notifications.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { screen, waitFor } from "@testing-library/vue";
+import { expect } from "vitest";
+import { test, describe } from "../../drivers/vitest/driver";
+import * as precondition from "../../preconditions";
+import { getTrialBar } from "./questions/trialLicenseBar";
+
+describe("FEATURE: Trial license notifications", () => {
+  describe("RULE: The user should know it is using a trial license at all times", () => {
```
Comment (cquirosj) [inline src/Frontend/test/specs/licensing/trial-license-notifications.spec.ts:8]:
```suggestion
  describe("RULE: The user should know they is using a trial license at all times", () => {
```

**Review by cquirosj — APPROVED** [review]

**Review by WilliamBZA — APPROVED** [review]

**Pull request merged**